contracts/deploy/phase0.sol

```diff
@@ -13,7 +13,6 @@ contract Phase0 {
     RadicleToken public immutable token;
```

Comment:
Code is correct, but commit message isn't ;)

Reply:
What do you mean?

Reply:
It says "Remove timelock deployment"

Reply:
Yes and the commit with that message does exactly that :thinking: Anyway we shouldn't sweat too much over any commit messages except the first one, they will be squashed before merging.

Reply:
The commit removes the treasury, not the timelock :grimacing:

Reply:
Right :facepalm: